include/mbedtls/cmac.h

```diff
@@ -35,104 +36,118 @@ extern "C" {
 #define MBEDTLS_DES3_BLOCK_SIZE         8
 
 #if defined(MBEDTLS_AES_C)
-#define MBEDTLS_CIPHER_BLKSIZE_MAX      16  /* longest used by CMAC is AES */
+#define MBEDTLS_CIPHER_BLKSIZE_MAX      16  /* The longest block used by CMAC is AES. */
 #else
-#define MBEDTLS_CIPHER_BLKSIZE_MAX      8   /* longest used by CMAC is 3DES */
+#define MBEDTLS_CIPHER_BLKSIZE_MAX      8   /* The longest block used by CMAC is 3DES. */
 #endif
 
 #if !defined(MBEDTLS_CMAC_ALT)
 
 /**
- * CMAC context structure - Contains internal state information only
+ * The CMAC context structure. Contains internal-state information only.
```

Comment:
I don't like the pre-existing wording very much and would like to improve it while we're at it. I'd prefer a plain `The CMAC context structure.` as for other structures, or `The CMAC context structure. Contains internal information only and should not be accessed directly.`, or `The internal CMAC context structure. This should not be used directly.` The structure does not occur in any public API and is only visible in the header because `cipher.c` needs to know its size.

Reply:
Accepted and standardized this to: "The CMAC context structure."